ktest/linux/slab.h

```diff
@@ -35,6 +35,7 @@
 /* asm/page.h */
 #define PAGE_SIZE	4096UL
 #define PAGE_MASK	(~(PAGE_SIZE-1))
+#define PAGE_SHIFT	12
```

Comment:
Why you don't use all this values from linux kenrel header?

Reply:
It's library to override all these values from linux kernel and use it in userspace.